com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/contribution/ILanguageContribution.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.contribution;
+
+import com.avaloq.tools.ddk.xtext.modelinference.IAdditionalModelInferrer;
+import com.avaloq.tools.ddk.xtext.resource.IAdditionalExport;
+
+
+/**
+ * Extends the specified language.
+ */
+public interface ILanguageContribution {

```

Comment:
Detail: I notice we use both the terms "contribution" and "addition" in the type and method names and in the documentation we even often talk about "extension". Maybe we could use only one term? 

ILanguageExtensions? ILanguageExtensionServices? IModelInferrerContribution? IModelInferrerExtension?

In any case this interface here should use a plural form like "contributions" or "extensions".

Reply:
Right, plural form should be used here. Naming, yes... a bit too much. Will think.